src/pages/HomePage.tsx

```diff
@@ -1,20 +1,242 @@
+      releaseDate: "2021-12-15",
+      voteCount: 8569,
+    },
+    {
+      id: 14,
+      title: "Euphoria",
+      posterPath:
+        "https://www.themoviedb.org/t/p/w220_and_h330_face/3Q0hd3heuWwDWpwcDkhQOA6TYWI.jpg",
+      releaseDate: "2021-12-15",
+      voteCount: 8569,
+    },
+    {
+      id: 15,
+      title: "Euphoria",
+      posterPath:
+        "https://www.themoviedb.org/t/p/w220_and_h330_face/3Q0hd3heuWwDWpwcDkhQOA6TYWI.jpg",
+      releaseDate: "2021-12-15",
+      voteCount: 8569,
+    },
+  ],
+  onTv: [
+    {
+      id: 1,
+      title: "Hura hura",
+      posterPath:
+        "https://www.themoviedb.org/t/p/w220_and_h330_face/3Q0hd3heuWwDWpwcDkhQOA6TYWI.jpg",
+      releaseDate: "2021-12-15",
+      voteCount: 8569,
+    },
+    {
+      id: 2,
+      title: "Hura Hura",
+      posterPath:
+        "https://www.themoviedb.org/t/p/w220_and_h330_face/3Q0hd3heuWwDWpwcDkhQOA6TYWI.jpg",
+      releaseDate: "2021-12-15",
+      voteCount: 8569,
+    },
+    {
+      id: 3,
+      title: "Hura Hura",
+      posterPath:
+        "https://www.themoviedb.org/t/p/w220_and_h330_face/3Q0hd3heuWwDWpwcDkhQOA6TYWI.jpg",
+      releaseDate: "2021-12-15",
+      voteCount: 8569,
+    },
+    {
+      id: 4,
+      title: "Hura Hura",
+      posterPath:
+        "https://www.themoviedb.org/t/p/w220_and_h330_face/3Q0hd3heuWwDWpwcDkhQOA6TYWI.jpg",
+      releaseDate: "2021-12-15",
+      voteCount: 8569,
+    },
+    {
+      id: 5,
+      title: "Hura Hura",
+      posterPath:
+        "https://www.themoviedb.org/t/p/w220_and_h330_face/3Q0hd3heuWwDWpwcDkhQOA6TYWI.jpg",
+      releaseDate: "2021-12-15",
+      voteCount: 8569,
+    },
+    {
+      id: 6,
+      title: "Hura Hura",
+      posterPath:
+        "https://www.themoviedb.org/t/p/w220_and_h330_face/3Q0hd3heuWwDWpwcDkhQOA6TYWI.jpg",
+      releaseDate: "2021-12-15",
+      voteCount: 8569,
+    },
+    {
+      id: 7,
+      title: "Hura Hura",
+      posterPath:
+        "https://www.themoviedb.org/t/p/w220_and_h330_face/3Q0hd3heuWwDWpwcDkhQOA6TYWI.jpg",
+      releaseDate: "2021-12-15",
+      voteCount: 8569,
+    },
+    {
+      id: 8,
+      title: "Hura Hura",
+      posterPath:
+        "https://www.themoviedb.org/t/p/w220_and_h330_face/3Q0hd3heuWwDWpwcDkhQOA6TYWI.jpg",
+      releaseDate: "2021-12-15",
+      voteCount: 8569,
+    },
+    {
+      id: 9,
+      title: "Hura Hura",
+      posterPath:
+        "https://www.themoviedb.org/t/p/w220_and_h330_face/3Q0hd3heuWwDWpwcDkhQOA6TYWI.jpg",
+      releaseDate: "2021-12-15",
+      voteCount: 8569,
+    },
+    {
+      id: 10,
+      title: "Hura Hura",
+      posterPath:
+        "https://www.themoviedb.org/t/p/w220_and_h330_face/3Q0hd3heuWwDWpwcDkhQOA6TYWI.jpg",
+      releaseDate: "2021-12-15",
+      voteCount: 8569,
+    },
+  ],
+};
+
+const tabOptions = [
+  { key: "streaming", title: "Streaming" },
+  { key: "onTv", title: "On TV" },
+];
+
+const titleSlider = {
+  title: "Sedang Populer",
+};
```

Comment:
ini langsung diganti title aja biar gak bingung

```suggestion
const title = "Sedang Populer";
```

Reply:
Oooo jadi ga harus dijadikan objek yaa, soalnya hanya satu

Reply:
yes sure